code/modules/power/apc.dm

```diff
@@ -469,7 +473,7 @@
 			to_chat(user, "<span class='warning'>Nothing happens.</span>")
 		else
 			flick("apc-spark", src)
-			if(do_after(user, 6, TRUE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
Yea, it's a not-so-stealthy emagging, plus the delay is very short.